mule-migration-tool-e2e-tests/src/test/resources/e2e/integration/graph_cycles3/input/src/main/app/mule-config.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:http="http://www.mulesoft.org/schema/mule/http" xmlns="http://www.mulesoft.org/schema/mule/core" xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+      xmlns:file="http://www.mulesoft.org/schema/mule/file"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+http://www.mulesoft.org/schema/mule/file http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd">
+    <http:listener-config name="HTTP_Listener_Configuration" doc:name="HTTP Listener Configuration" host="0.0.0.0" port="8081" />
+    <file:connector name="File" outputPattern="test123.txt" autoDelete="true" streaming="true" validateConnections="true" />
+    <sub-flow name="flow3">
+        <logger/>
+    </sub-flow>
+    <sub-flow name="flow2">
+        <set-property propertyName="http.statusCode" value="202"/>
```

Comment:
should be `http.status` if expected to be set as status code

Reply:
changed in last rev